pkg/networkmanager/interfaces.js

```diff
@@ -720,13 +720,25 @@ export function NetworkManagerModel() {
         case 11: return 'vlan';
         case 12: return 'adsl';
         case 13: return 'bridge';
-        case 14: return 'loopback';
+        case 14: return 'generic';
         case 15: return 'team';
         case 16: return 'tun';
         case 17: return 'ip_tunnel';
         case 18: return 'macvlan';
         case 19: return 'vxlan';
         case 20: return 'veth';
+        case 21: return 'macsec';
+        case 22: return 'dummy';
+        case 23: return 'ppp';
+        case 24: return 'ovs_interface';
+        case 25: return 'ovs_port';
+        case 26: return 'ovs_bridge';
+        case 27: return 'wpan';
+        case 28: return '6lowpan';
+        case 29: return 'wireguard';
+        case 30: return 'wifi_p2p';
+        case 31: return 'vrf';
```

Comment:
These 11 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18850-20230526-094431-7eb31071-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)